app/extensions/submission/__init__.py

```diff
@@ -238,6 +238,11 @@ def ensure_repository(self, submission, remote=True):
         with open(submission_metadata_path, 'w') as submission_metadata_file:
             json.dump(submission_metadata, submission_metadata_file)
 
+        # gitignore_path = os.path.join(submission_path, '.gitignore')
```

Comment:
If this code is not needed, let's delete it

Reply:
yeah will remove for now.  was hoping to get a `.gitignore` in there to keep some stuff out of repo.  but it didnt work.  meh.  later.